modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+[id="adding-{product-title}-container-images_{context}"]
+= Adding the {product-title} container images
+
+You can embed {product-title}'s container images into the {op-system-ostree} images so that they are immediately available to the CRI-O container engine after booting. By embedding, images are not pulled over the network from a container registry. In Image Builder, a container image is embedded by adding a reference to it to the Image Builder blueprint.
+
+.Example adding a container image to a blueprint:
```

Comment:
```suggestion
.Example adding a container image to a blueprint
```

Reply:
Is this the output example from running the `jq` command?

I am not sure what this is referring to or how I should run this terminal command. 

Maybe this could be rephrased as "Example blueprint syntax"? For example: 
![image](https://user-images.githubusercontent.com/82119979/211929396-6fea6ce7-0759-460a-80cf-e727d44d6982.png)

Reply:
This is trying to explain the syntax for adding a configuration section to the blueprint file for the user to add their own container image to be embedded in the OS image. There is no command being run in this part of the explanation.

Reply:
In that case, I think labelling it `.Example blueprint file` or `.Blueprint file container image syntax` would work. Blueprint files use the TOML format, correct?